### PR TITLE
Fix typos and links in docs

### LINKED ISSUE #4383: Doc edit review
**Some minor typos/broken links/other in following files:** 

https://github.com/projectcontour/contour/blob/main/SITE_CONTRIBUTION.md#
typo

https://projectcontour.io/resources/philosophy/
typo and needs link to code of conduct

https://projectcontour.io/resources/how-we-work/
typos

https://projectcontour.io/docs/v1.20.0/start-contributing/
references to office hours are out of date, redirect some office hours links to community page

https://projectcontour.io/
new issue link at bottom of page is incorrect

https://projectcontour.io/resources/support/
checkmarks in table are not displaying

https://projectcontour.io/resources/security-threat-model/
typo

https://projectcontour.io/resources/faq/
typo

### PR DESCRIPTION
**Pull request by kcoriordan**
First PR!
Edit review of site, fixes typos, links and other minor display issues. 
Fixes #4383

### CODE REVIEW

File: site/content/resources/security-threat-model.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ We maintain a CI pipeline that runs golangci-lint including the usual set of Go
 ## Conclusion
 
 The Contour team works hard to understand the project's security context and keep up with the state of the art for Kubernetes security.
-The team hopes that this examination of our security model provides some insight into both how we develop Contour and what adminsitrators should be thinking about to run Contour in as secure way as possible.
+The team hopes that this examination of our security model provides some insight into both how we develop Contour and what administrators should be thinking about to run Contour in as secure way as possible.
```
Comment (skriss) [inline site/content/resources/security-threat-model.md:98]: This one got fixed in https://github.com/projectcontour/contour/pull/4382 (just merged), but I think GitHub will handle it fine.

File: site/themes/contour/layouts/partials/team.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="team">
     <div class="wrapper">
         <h2>The Contour Team</h2>
-        <p>Contour is released as open source software and provides community support through our GitHub project page. If you encounter an issue or have a question, feel free to reach out on the <a href="#">GitHub issues page for Contour</a>.</p>
+        <p>Contour is released as open source software and provides community support through our GitHub project page. If you encounter an issue or have a question, feel free to reach out on the <a href="https://github.com/vmware-tanzu/community-edition/issues/new/choose">GitHub issues page for Contour</a>.</p>
```
Comment (skriss) [inline site/themes/contour/layouts/partials/team.html:4]:
```suggestion
        <p>Contour is released as open source software and provides community support through our GitHub project page. If you encounter an issue or have a question, feel free to reach out on the <a href="https://github.com/projectcontour/contour/issues/new/choose">GitHub issues page for Contour</a>.</p>
```

**Review by skriss — CHANGES_REQUESTED** [review]
Thanks @kcoriordan, looks great! Just a couple minor comments.

**Review by skriss — APPROVED** [review]
LGTM

**Comment by kcoriordan** [comment]
@skriss - not sure if this is ok, looks like some other commits from PR #4379 got picked up, not sure how this happened, not sure if this is ok to merege

File: site/themes/contour/layouts/partials/team.html (hunk 2 above)
Comment (skriss) [reply site/themes/contour/layouts/partials/team.html:4]: Oops, looks like the fix for this got reverted in the latest commit, can you reapply the fix?

**Comment by kcoriordan** [comment]
thanks @skriss for your help :)

**Comment by skriss** [comment]
No problem, 🎉   for first Contour PR!

**Pull request merged**